LuckySpin/Services/SpinService.cs

```diff
@@ -15,11 +15,18 @@ public SpinService(ISpinRepository sr)
 
         }
 
-
-        public double CalculateAvgWins()
+        public double CalculateAvgWins(bool winning)
         {
-            //TODO: Write logic to use the "real" spinRepository NOT the test data
-            return .1; 
+            //TODO: Write logic to use the "real" spinRepository NOT the test data'
+            double wins = 0;
+            if (winning)
+                wins = 1;
+            foreach (Spin s in spinRepository.GetSpins())
+            {
+                if (s.IsWinning == true)
+                    wins++;
+            }
```

Comment:
Consider using the Lamda Extension method Count() on the spins collection returned by GetSpins()